vendor/github.com/ladydascalie/currency/.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Why not add it to the global `.gitignore`? It looks more practical.

Reply:
Hmm this file was part of the repo, not something I manually created, so I'd better leave it as-is

Reply:
I missed that its part of the external library 🤦‍♂️